src/filter/RuleHelper.js

```diff
@@ -650,6 +667,38 @@ const exports = class {
     return filter;
   }
 
+  /**
+   * Create and return an OpenLayers filter object using the available
+   * dimensions filters configuration within the data source.
+   * @param {ngeo.DataSource} dataSource Data source from which to create the
+   *     filter.
+   * @return {?ol.format.filter.Filter} Filter
```

Comment:
A filter or null.

Reply:
Yes, like createFilterFromRule_ and createTimeFilterFromDataSource_.
Is there something to change in annotations ?